toolchain/sem_ir/ids.h

```diff
@@ -86,6 +86,20 @@ constexpr InstId InstId::Invalid = InstId(InvalidIndex);
       InstId::ForBuiltin(BuiltinInstKind::Name);
 #include "toolchain/sem_ir/builtin_inst_kind.def"
 
+// An ID of an instruction that is referenced absolutely within a typed
```

Comment:
The phrasing of this comment makes it sound kind of like "absoluteness" is a property of the inst itself, but based on the code, I gather it's actually a property of the ID. In other words, in principle you could have an `AbsoluteInstId` and an `InstId` that refer to the same inst. Is that right?

Reply:
Yes, that's right. I've updated the comment to try to make this clearer.